state/validation.go

```diff
@@ -48,26 +48,13 @@ func validateBlock(evidencePool EvidencePool, stateDB dbm.DB, state State, block
 		)
 	}
 
-	// Validate app info
```

Comment:
add a note at the beginning that `appHash` and `ResultHash` will be checked in `CheckBlock`